configure

```diff
@@ -1599,6 +1602,9 @@ Optional Packages:
   --with-experimental-isolated-subinterpreters
                           better isolate subinterpreters, experimental build
                           mode (default is no)
+  --without-static-libpython
+                          do not build libpythonMAJOR.MINOR.a and do not
+                          install python.o (default is yes)
```

Comment:
it's unclear if "yes" means "yes with static" or "yes without static". to me at least.